apps/etl/transform/sources/emdat.py

```diff
@@ -32,3 +41,60 @@ def transform_emdat_data(extraction_id, **kwargs):
         extraction_id,
         json_data,
     )
+
+
+@shared_task
+def transform_data(source, transformer, data_source, extraction_id, data):
+    logger.info(f"Transformation started for {source} data")
+    ext_instance = ExtractionData.objects.get(id=extraction_id)
+
+    # create transform object
+    transform_obj = Transform.objects.create(
+        extraction=ext_instance,
+        status=Transform.Status.PENDING,
+    )
+
+    # initialize bulk manager to create the PyStacLoadData in bulk.
+    bulk_mgr = BulkCreateManager(chunk_size=1000)
+
+    # XXX: Local file is used
+    # geocoder = GAULGeocoder(gpkg_path="/code/gaul2014_2015.gpkg")
+    geocoder = GAULGeocoder(service_base_url=settings.GEOCODER_URL)
```

Comment:
Use if condition to tackle both conditions to use either local file or service_url.